recipe/windows-bat.patch

```diff
@@ -0,0 +1,11 @@
+--- graphviz/backend.py
++++ graphviz/backend.py
+@@ -114,6 +114,8 @@
+     suffix = '.'.join(reversed(format_arg))
+     format_arg = ':'.join(format_arg)
+

```

Comment:
This needs one space before the line break.

```
patch: **** malformed patch at line 6: 

patching file graphviz/backend.py
```

ref: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=41373